src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -514,7 +514,8 @@ public void testStaticImportReception() throws Exception {
         checkConfig.addAttribute("separated", "true");
         checkConfig.addAttribute("groups", "java, org");
         final String[] expected = {
-            "4: " + getCheckMessage(MSG_SEPARATION, "org.antlr.v4.runtime.CommonToken.*"),
+            // Till https://github.com/checkstyle/checkstyle/issues/5279
+            //"4: " + getCheckMessage(MSG_SEPARATION, "org.antlr.v4.runtime.CommonToken.*"),
```

Comment:
this is good point, thanks for commenting it.

But missed violation (false-negative) is ok for some time. False-positives are blockers for users who have ERROR severity in their builds.

Reply:
The test input is
```
import static java.lang.Math.*;
import static org.antlr.v4.runtime.CommonToken.*;
import static org.antlr.v4.runtime.CommonToken.*;  //Repetitive static import
```
The code in the master branch demands a gap between the groups. We do not have the property `separatedStaticGroups` yet, so this violation is commented out till #5279